static/src/javascripts/projects/common/modules/article/open-module.js

```diff
@@ -31,23 +33,21 @@ define([
     return {
         init: function () {
             if (config.page.openModule) {
-                spacefinder.getParaWithSpace(getSpacefinderRules()).then(function (space) {
-                    if (space) {
-                        ajax({
-                            url: config.page.openModule,
-                            crossOrigin: true,
-                            method: 'get'
-                        }).then(function (resp) {
-                            if (resp.html) {
-                                fastdom.write(function () {
-                                    $.create(resp.html)
-                                        .addClass('element--supporting')
-                                        .insertBefore(space);
-                                    $('.submeta-container--break').removeClass('submeta-container--break');
-                                });
-                            }
-                        });
-                    }
+                spaceFiller.fillSpace(getSpacefinderRules(), function (spaces) {
+                    ajax({
+                        url: config.page.openModule,
+                        crossOrigin: true,
+                        method: 'get'
+                    }).then(function (resp) {
+                        if (resp.html) {
```

Comment:
When wouldn't there be a `resp.html`? Do we need to throw an error if there isn't one? (Silent errors are the worst.)